monitor_packages/SQLServer/README.md

```diff
@@ -0,0 +1,17 @@
+# sumologic-sqlserver-monitors
+
+This Terraform script configures pre-packaged Sumo Logic Monitors for SQL Server using Terraform modules.
+For installation and configuration, please look at Sumo Logic SQL Server [Help Document](https://help.sumologic.com/07Sumo-Logic-Apps/04Microsoft-and-Azure/Microsoft_SQL_Server).
```

Comment:
The link should point to the install alerts, dashboards page, i.e. https://help.sumologic.com/07Sumo-Logic-Apps/04Microsoft-and-Azure/Microsoft_SQL_Server/Microsoft-SQL-Server-App-Dashboards

Reply:
@langmtt